OrbitGl/ThreadStateTrack.cpp

```diff
@@ -97,6 +97,33 @@ static std::string GetThreadStateName(ThreadStateSliceInfo::ThreadState state) {
   }
 }
 
+static std::string GetThreadStateDescription(ThreadStateSliceInfo::ThreadState state) {
+  switch (state) {
+    case ThreadStateSliceInfo::kRunning:
+      return "The thread is currently scheduled on the CPU.";
+    case ThreadStateSliceInfo::kRunnable:
+      return "The thread is ready to use the CPU, but is currently not scheduled.";
+    case ThreadStateSliceInfo::kInterruptibleSleep:
+      return "The thread is waiting for a resource to become available or for an event to happen.";
+    case ThreadStateSliceInfo::kUninterruptibleSleep:
+      return "The thread performed a specific system call that cannot be interrupted by any signal "
+             "and is waiting for the call to complete.";
+    case ThreadStateSliceInfo::kStopped:
+      return "The execution of the thread was suspended with the SIGSTOP signal.";
+    case ThreadStateSliceInfo::kTraced:
+      return "The thread is stopped because a tracer (for example, a debugger) is attached to it.";
+    case ThreadStateSliceInfo::kDead:
```

Comment:
Indicate the intentional fallthrough here. The style guide suggests `ABSL_FALLTHROUGH_INTENDED;` (https://google.github.io/styleguide/cppguide.html#Loops_and_Switch_Statements).
I guess `[[fallthrough]];` should be supported as well by our compilers.

Reply:
Good point, there were also a couple missing in a function above, done.